tool/format.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
I think it might be nice to start using [melos](https://melos.invertase.dev) soon to make handling the different packages a bit easier and faster, I'll give it some more thought and put it in the ideas 👌

Reply:
Never worked with it before. Can you help with it?